Classes/Command/NamespaceCommandController.php

```diff
@@ -86,11 +84,10 @@ protected function getDeprecatedFileTemplate()
     {
         return <<<'EOD'
 <?php
-namespace {{oldNamespace}}
+namespace {{oldNamespace}};
 
 /*
- * This class namespace was deprecated and moved to {{newNamespace}},
- * find the replacement here: {{newPhpFile}}
+ * This class namespace was deprecated and moved to the new PSR-4 namespace {{newNamespace}}
```

Comment:
ah yeah, misleading. But need to discuss if we those old files at all. Main issue here is that the absolute path (/Users/...) is set